Makefile

```diff
@@ -53,13 +53,14 @@ clean_hard: clean
 # Compiles sass to css
 css:
 	@mkdir -p $(PUBLIC_FOLDER)/stylesheets
-	@$(NODE_SASS) --output-style compressed -o $(PUBLIC_FOLDER)/stylesheets $(STYLESHEETS_LOC)
-	@$(POSTCSS) -r $(PUBLIC_FOLDER)/stylesheets/* --no-map
+	@$(NODE_SASS) --output-style compressed -o $(PUBLIC_FOLDER)/stylesheets $(STYLESHEETS_LOC) --source-map true
+	@$(POSTCSS) -r $(PUBLIC_FOLDER)/stylesheets/*.css --no-map
```

Comment:
You'll need to implement sourcemaps further - PostCSS will manipulate the output files but won't regenerate the sourcemap, so you'll get the wrong line numbers in this scenario.